src/app/templates/certificate-geek/certificate-gci18.component.html

```diff
@@ -5,16 +5,15 @@
     <h1 class="org_name">Google Code In</h1>
     <p class="coa">Certificate of Achievement</p>
     <p class="pre_text">Presented to:</p>
-    <p class="candidate-name">Name</p>
-    <h3 class="desc">Description</h3>
+    <p class="candidate-name">{{ this.certificateForm.value.issued_to }}</p>
+    <h3 class="desc">{{ this.certificateForm.value.issued_for }}</h3>
       <div class="signature-area">
         <h4 class="signHead" align="left">Sign</h4>
         <h4 class="sign" align="left"></h4>
       </div>
       <div class="date-area">
         <h4 class="dateHead" align="right">Date</h4>
-        <h4 class="date" align="right"></h4>
+        <h4 class="date" align="right">{{ this.certificateForm.value.date }}</h4>
       </div>
     </div>
-  </div>
-</div>
\ No newline at end of file
+  </div>
```

Comment:
A newline at the end is a must.

Reply:
I've added a newline, but I am new to this. will you please explain to me why a newline at the end is a must?

Reply:
Check [here](https://unix.stackexchange.com/a/18789/281129).

Basically, that's the definition of [line](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) as per POSIX.

In addition, the [editorconfig](https://github.com/JBossOutreach/certificate-generator-front/blob/master/.editorconfig) that defines, how your GitHub Editor will show code is set up to parse a newline at the end, else it shows error.

**PS**: One of important reason is it helps compiler to separate imports from code.